### PR TITLE
LPS-116680 [Testing] Add test for IndividualSegmentsMessageListener

### PR DESCRIPTION
**Pull request by darquesdev**
**Motivation**

Add missed unit test for IndividualSegmentsMessageListener

### CODE REVIEW

**Comment by darquesdev** [comment]
ci:test:sf

**Comment by darquesdev** [comment]
ci:test:relevant

**Comment by darquesdev** [comment]
@cgoncas the review is not needed yet

File: modules/dxp/apps/segments/segments-asah-connector/src/test/java/com/liferay/segments/asah/connector/internal/messaging/IndividualSegmentsMessageListenerTest.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+	@Test
+	public void testDoReceiveWithEmptyAcClientUserId() throws Exception {
+		Message message = new Message();
+
+		message.put("userId", StringPool.BLANK);
+
+		_individualSegmentsMessageListener.doReceive(message);
+
+		Mockito.verify(
```
Comment (cgoncas) [inline modules/dxp/apps/segments/segments-asah-connector/src/test/java/com/liferay/segments/asah/connector/internal/messaging/IndividualSegmentsMessageListenerTest.java:67]: We should not create test like: 

`Mockito.verify(
			_individualSegmentsChecker, Mockito.times(1)
		).checkIndividualSegments(
			message.getLong("companyId"), message.getString("userId")
		);`

This test is highly coupled to the implementation. here there is a nice chapter that explains best practices with Mocks in unit testing: https://livebook.manning.com/book/unit-testing/chapter-9/v-3/5

Comment (cgoncas) [reply modules/dxp/apps/segments/segments-asah-connector/src/test/java/com/liferay/segments/asah/connector/internal/messaging/IndividualSegmentsMessageListenerTest.java:67]: In general, a test always should follow the steps:

1. Given an initial condition (this step is optional)
2. When something happens
3. A result is expected.

**Comment by darquesdev** [comment]
this task was discarded https://issues.liferay.com/browse/LPS-116680

**Comment by darquesdev** [comment]
this task was discarded https://issues.liferay.com/browse/LPS-116680

**Pull request closed without merging**